app/models/waste_exemptions_engine/address.rb

```diff
@@ -4,10 +4,6 @@
 
 module WasteExemptionsEngine
   class Address
-    scope :search_for_postcode, lambda { |term|
-      where("UPPER(postcode) LIKE ?", "%#{term&.upcase}%")
-    }
-
-    scope :site, -> { where(address_type: 3) }
+    include WasteExemptionsEngine::CanBeSearchedLikeAddress
```

Comment:
I don't think it needs the Module name

```suggestion
    include CanBeSearchedLikeAddress
```

I could be wrong 😁

Reply:
You are correct! I got it right for the others so not sure why I did it differently here 🤔